finance/jobs/2020_githubresponder.rst

```diff
@@ -0,0 +1,25 @@
+GitHub First/Last Responder
+===========================
+
+The Astropy Project is looking to hire a Github first responder / last responder. The overarching goal of this position is to improve the first time contributor experience, reduce the technical barriers to contributions, reduce the number of open issues, and increase the number of merged pull requests. This is not intended to cover all issues and PRs opened in astropy, but rather to support and lighten the load on the rest of the volunteer community.
+An ideal candidate would be familiar with the astropy project ecosystem and capabilities, comfortable with the Github Development workflow, and have experience providing supportive feedback and technical advice in online environments.
+
+This role includes  monitoring  the astropy issue tracker, add labels to new issues, provide a friendly welcome message on issues and pull requests, and give starting points for how to resolve the issue or move the pull request forward. The “last responder” aspect of the role includes helping to close out issues and PRs, helping with rebasing, and troubleshooting failing tests.
+
+We expect a commitment of about 8-10 hours/week for 12 months. The Astropy Project can pay inside or outside of the US. Compensation could be provided as a consulting fee or as a subaward. This is a remote-only position.
+
+Please send a short (one paragraph) email to finance@astropy.org with a description of your interest in the role, your qualifications, and any relevant experience.  If possible, please provide links to a few examples of issues/PRs where you have been involved as a fist and/or last responder.
+
+The Astropy project is committed to fostering an inclusive community. The community of participants in open source Astronomy projects is made up of members from around the globe with a diverse set of skills, personalities, and experiences. It is through these differences that our community experiences success and continued growth. See https://www.astropy.org/code_of_conduct.html for details.
+
+The Astropy Project specfically encourages underrepresented parts of the community to apply.
+
+All applications received less than four weeks after this PR is merged will receive full considerations, but you can apply until the position is filled.
```

Comment:
Maybe put the exact date and time (with timezone) here right before merge. Otherwise, people need to constantly check the PR timestamp.

Reply:
That's the plan.